FilmFestivalLoader/IDFA/parse_idfa_html.py

```diff
@@ -603,30 +859,47 @@ def __lt__(self, other):
         return self.sortstring < other.sortstring
 
 
-class Screening(planner.Screening):
-    def __init__(self, film, screen, start_datetime, end_datetime, qa, extra, audience, combination_program_url):
-        planner.Screening.__init__(self, film, screen, start_datetime, end_datetime, qa, extra, audience)
-        self.combination_program_url = combination_program_url
+class IdfaScreening(Screening):
+    def __init__(self, film, screen, start_datetime, end_datetime,
+                 qa='', extra='', audience=None, combi_title=None, combi_url=None):
+        super().__init__(film, screen, start_datetime, end_datetime, qa, extra, audience)
+        self.combi_title = combi_title
+        self.combi_url = combi_url
 
 
-class IdfaData(planner.FestivalData):
+class IdfaData(FestivalData):
+    duplicates_by_screening = {}
 
     def __init__(self, directory):
-        planner.FestivalData.__init__(self, directory, festival_city)
+        super().__init__(directory, festival_city)
         self.compilation_by_url = {}
 
     def film_key(self, title, url):
         return url
 
+    def screening_can_go_to_planner(self, screening):
+        can_go = screening.is_public()
+        if can_go:
+            try:
+                self.duplicates_by_screening[screening] += 1
+                can_go = False
+            except KeyError:
+                self.duplicates_by_screening[screening] = 0
+        if can_go:
+            can_go = screening.combi_url is None
+        if can_go:
+            can_go = screening.screen.screen_id != 126      # de Brakke Grond
+        if can_go:
+            can_go = not self.is_coinciding(screening)
+        return can_go
+
+    def film_can_go_to_planner(self, film_id):
+        return True
+
     def is_coinciding(self, screening):
         # Get the film info.
         film_info = screening.film.film_info(self)
 
-        # Check if the film is a combination program.
-        screened_films = film_info.screened_films
-        if len(screened_films):
-            return False
```

Comment:
Please consider to put this code back, as to prevent that a costly operation is tries for every screening.

Reply:
Done.